sphinx/source/docs/dev_guide/bokehjs.rst

```diff
@@ -43,113 +43,14 @@ web workers, asm.js, SIMD, and parallel JS (e.g. River Trail).
 Interface
 ---------
 
-.. warning::
-    While it is possible in principle to construct plots using only BokehJS
-    models directly, this would be somewhat tedious. Work is ongoing to create
-    higher level integrations and interfaces for BokehJS.
-
 BokehJS is a standalone JavaScript library for dynamic and interactive
 visualization in the browser. It is built on top of HTML5 canvas, and designed
 for high-performance rendering of larger data sets. Its interface is declarative,
 in the style of Protovis_, but its implementation consists of a reactive scene
-graph (similar to Chaco_). Some examples for different types of plots are shown
-below in `devguide_bokehjs_examples`_.
+graph (similar to Chaco_).
 
 The full BokehJS interface is described detail in :doc:`../reference/bokehjs`
 
-.. _devguide_bokehjs_dependencies:
-
-Dependencies
-------------
-
-BokehJS ships with all of its vendor dependencies built in. For reference, the
-vendor libraries that BokehJS includes are:
-
-* almond
-* backbone-amd
-* bootstrap-3.1.1
-* font-awesome-4.2.0
-* gear-utils
-* hammer.js-2.0.4
-* jqrangeslider-5.7.0
-* jquery-1.11.1
-* jquery-event-2.2
-* jquery-mousewheel-3.1.12
-* jquery-ui-1.11.2
-* jsnlog.js-2.7.5
-* kiwi
-* numbro.js-1.6.2
-* qunit
-* rbush
-* requirejs
-* slick-grid-2.1.0
-* sprintf
-* text
-* timezone
-* underscore-amd
-
-.. _devguide_bokehjs_examples:
-
-Examples
---------
-
-.. warning::
-    These examples currently use an old, deprecated engineering JS interface
-    that has since been removed.  Work is ongoing to create a new interface
-    and update these examples.
-
-Several live examples that demonstrate the BokehJS interface are available on
-JSFiddle_. Click on "CoffeeScript" to see the code that generates these plots,
-or on "Edit in JSFiddle" to fork and create your own examples.
-
-Scatter
-~~~~~~~
-
-This example shows a scatter plot where every circle has its own radius and
-color.
-
-.. raw:: html
-
-    <iframe
-        width="100%"
-        height="700"
-        src="http://jsfiddle.net/bokeh/Tw5Sm/embedded/result,js/"
-        allowfullscreen="allowfullscreen"
-        frameborder="0"
-    ></iframe>
-
-Lorenz
-~~~~~~
-
-This example shows a 2D projection of the Lorenz attractor. Sections of the
-line are color-coded by time.
-
-.. raw:: html
-
-    <iframe
-        width="100%"
-        height="700"
-        src="http://jsfiddle.net/bokeh/s2k59/embedded/result,js"
-        allowfullscreen="allowfullscreen"
-        frameborder="0"
-    ></iframe>
-
-Animated
-~~~~~~~~
-
-This example shows how it it possible to animate BokehJS plots by updating
-the data source.
-
-.. raw:: html
-
-    <iframe
-        width="100%"
-        height="700"
-        src="http://jsfiddle.net/bokeh/K8P4P/embedded/result,js/"
-        allowfullscreen="allowfullscreen"
-        frameborder="0"
-    ></iframe>
-
```

Comment:
Intend to add more information back here as part of separate PR to document new BokehJS interface.